### ISSUE TITLE
Rename to switchboard-plug-desktop

### ISSUE THREAD

**Issue by peteruithoven** [elementary/switchboard-plug-pantheon-shell#146]
Shouldn't this plug be renamed to "switchboard-plug-desktop", making it more consistent with it's name in System Settings?

**Comment by danirabbit** [comment]
This is fixed in code now, only the github page is still named `switchboard-plug-pantheon-shell`

**Comment by ryonakano** [comment]
Maybe it's worth revisiting this in OS 9?

- [ ] Change repository URL
- [ ] Change [Weblate project URL](https://l10n.elementary.io/projects/switchboard/switchboard-plug-pantheon-shell/) and review Git settings
- [ ] Replace old URLs in README, metainfo, and packaging
- [ ] (Optionally) Replace `PantheonShell` namespace in the source code

**Comment by danirabbit** [comment]
Yeah on launchpad I think most of the recipes have been renamed to `settings-*` as well. It's a lot less to type haha. It'll just be kind of a pain to update everything and make sure nothing breaks but it would be a lot nicer in the end
